docs/4.3/admin-guide.md

```diff
@@ -1565,8 +1565,7 @@ Teleport will handle its own SSL on top of that with its own certificates.
     balancer you'll need to run Teleport with `--insecure-no-tls`
 
 If your load balancer supports HTTP health checks, configure it to hit the
-`/webapi/ping` endpoint on the proxy. This endpoint will reply `200 OK` if the
-proxy is running without problems.
+`/readyz` [diagnostics endpoint](metrics_logs_reference.md) on the proxy. This endpoint will reply `{"status":"ok"}` if the proxy is running without problems.
```

Comment:
This doesn't work unless you run with `--diag-addr=0.0.0.0:3000` or something...

```
vagrant@teleport:~$ curl -ik https://localhost:3080/readyz
HTTP/1.1 404 Not Found
Content-Type: text/plain; charset=utf-8
X-Content-Type-Options: nosniff
Date: Wed, 09 Sep 2020 21:34:41 GMT
Content-Length: 19

404 page not found
```